staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -229,6 +229,7 @@ type fancyResponseWriterDelegator struct {
 }
 
 func (f *fancyResponseWriterDelegator) CloseNotify() <-chan bool {
+	//lint:ignore SA1019 backwards compatibility
 	return f.ResponseWriter.(http.CloseNotifier).CloseNotify()
```

Comment:
Should we also mark `fancyResponseWriterDelegator.CloseNotify()` as deprecated?

Reply:
Done. Also: marking as deprecated makes the lint warning unnecessary.